core/src/edu/asu/c3simulator/screens/BusinessCreationGuide.java

```diff
@@ -0,0 +1,218 @@
+				showScreen(fundingScreen);
+			}
+			else if (currentScreen == fundingScreen)
+			{
+				System.out.println("Transitioning to tasks Screen");
+				showScreen(tasksScreen);
+			}
+			else if (currentScreen == tasksScreen)
+			{
+				System.out.println("Transitioning to Home Screen");
+				// TODO: Transition to main hub
+				// TODO: pass user choices to data layer
+			}
+		}
+	}
+	
+	@SuppressWarnings("unused")
+	private Game game;
+	private Stage stage;
+	private Skin skin;
+	private BusinessCreationGuideScreens industryScreen, directionScreen, fundingScreen,
+			tasksScreen, currentScreen;
+	TextButton progressButton;
+	
+	public BusinessCreationGuide(Game game)
+	{
+		this.game = game;
+		this.stage = new Stage();
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		industryScreen = new IndustryScreen(stage); // IndustryScreen();
+		directionScreen = new BusinessDirectonScreen(stage);
+		fundingScreen = new FundingScreen(stage); // FundingScreen();
+		tasksScreen = new TasksScreen(stage); // TasksScreen();
+		
+		TextButton industryButton = new TextButton("Industry", skin);
+		TextButton directonButton = new TextButton("Directon", skin);
+		TextButton fundingButton = new TextButton("Funding", skin);
+		TextButton tasksButton = new TextButton("Tasks", skin);
+		
+		industryButton.addListener(new navigationButtonListener(industryScreen,
+				"Industry"));
+		directonButton.addListener(new navigationButtonListener(directionScreen,
+				"Directon"));
+		fundingButton.addListener(new navigationButtonListener(fundingScreen, "Funding"));
+		tasksButton.addListener(new navigationButtonListener(tasksScreen, "Tasks"));
+		
+		Table navigationButtons = new Table();
+		navigationButtons.add(industryButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(directonButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(fundingButton).fillX().padTop(10);
+		navigationButtons.row();
+		navigationButtons.add(tasksButton).fillX().padTop(10);
+		
+		navigationButtons.center();
+		
+		float navigationButtonsLeft = 0.05f * stage.getWidth();
+		float navigationButtonsBottom = (stage.getHeight() / 2)
+				+ (navigationButtons.getPrefHeight());
+		navigationButtons.setTransform(true);
+		navigationButtons.setPosition(navigationButtonsLeft, navigationButtonsBottom);
+		
+		progressButton = new TextButton("Continue >", skin);
+		progressButton.addListener(new progressButtonListener());
+		
+		progressButton.setPosition(stage.getWidth() - progressButton.getPrefWidth()
+				- (0.01f * stage.getWidth()), (0.01f * stage.getWidth()));
+		
+		// TODO add Corner Advisor
+		// TODO add Home Button
+		
+		stage.addActor(navigationButtons);
+		stage.addActor(progressButton);
+		
+		currentScreen = industryScreen;
+		showScreen(industryScreen);
+	}
+	
+	/**
+	 * This methods hides the current screen before displaying the new one and displays
+	 * the appropriate progress Button.
+	 * 
```

Comment:
This was one of the first classes I wrote and didn't know how to handle it properly, so I improvised. You are right, switching through game should have been used, and also the navigation panel class would be helpful in here. 
However, it is going to take time to correct it, so I decided to spend that time on other tasks instead since this one is currently working fine.